settings.gradle.kts

```diff
@@ -1 +1,8 @@
 rootProject.name = "mc-votifier-lib"
+
+pluginManagement {
+    repositories {
+        mavenCentral()
```

Comment:
For whatever reason, gradle was still searching jcenter for EG the kotlin plugin. JCenter was deprecated and put into read-only mode last year only to have its certificate expire a few hours ago. This explicitly removes jcenter from the list for plugin resolution.

See also: https://github.com/gradle/gradle/issues/22864